tutorials/shading/your_first_shader/what_are_shaders.rst

```diff
@@ -0,0 +1,111 @@
+.. _doc_what_are_shaders:
+
+What are shaders?
+=================
+
+Introduction
+------------
+
+So, you have decided to give shaders a try. You have likely heard that they can be used to 
+create interesting effects that run incredibly fast. You have also likely heard that they
+are terrifying. Both are true. 
+
+Shaders can be used to create a wide range of effects (in fact everything drawn in a modern
+rendering engine is done with shaders). 
+
+Writing shaders can also be very difficult for people unfamiliar with them. Godot tries to make writing
+shaders a little easier by exposing many useful built-in features and handling some of the
+lower-level initialization work for you. However, GLSL (the OpenGL Shading Language, which Godot uses)
+is still unintuitive and restricting, especially for users who are used to GDScript.
+
+But what are they?
+------------------
+
+Shaders are a special kind of program that runs on Graphics Processing Units (GPUs). All computers
+have some sort of GPU, either one integrated into their CPU or discrete (meaning it is a separate 
+hardware component). GPUs are especially useful for rendering because they are optimized for running
```

Comment:
I'd change it to "most computers", also explicitely say "graphics card" somwhere - e.g. "meaning it is a seperate hardware component, for example the typical graphics card" - or something. 

If you're new to this, graphics card may be a term you're (already) familiar with.

Reply:
I agree. Thanks!